packages/proto-signing/src/codec/cosmos/bank/v1beta1/bank.ts

```diff
@@ -0,0 +1,650 @@
+/* eslint-disable */
+import { Coin } from "../../../cosmos/base/v1beta1/coin";
+import { Writer, Reader } from "protobufjs/minimal";
```

Comment:
> ts-proto is 2 years old and still has the feel of a young project. I'm a little nervous about that.

This is definitely something I share. Adding on top of that, ts-proto is heavily dependent on ts-poet (a string manipulation tool for code generation) by the same author. There is a non-zero risk these packages gets unmaintained down the road.

On the plus side, these two packages are not that big.